common/src/java/org/apache/hadoop/hive/ql/log/PerfLogger.java

```diff
@@ -92,6 +94,8 @@ public class PerfLogger {
 
   protected final Map<String, Long> startTimes = new ConcurrentHashMap<>();
```

Comment:
Should we add some tests in TestPerfLogger, TestQueryDisplay, or somewhere else to ensure that the data structures are populated correctly when the query compiles?